Source/diablo.cpp

```diff
@@ -1464,7 +1464,7 @@ void DisableInputWndProc(UINT uMsg, WPARAM wParam, LPARAM lParam)
 	case DVL_WM_SYSKEYDOWN:
 	case DVL_WM_SYSCOMMAND:
 	case DVL_WM_MOUSEMOVE:
-		GetMousePos(lParam);
+		//GetMousePos(lParam);

```

Comment:
Probably a better solution is to split the cases:
```cpp
	case DVL_WM_SYSCOMMAND:
		return;
	case DVL_WM_MOUSEMOVE:
		GetMousePos(lParam);
		return;
```

Reply:
I have now implemented this, thanks for figuring out where the issue where.
https://github.com/diasurgical/devilutionX/commit/c5b407f7a69b8d8af6a0debb3f187db47bdeaba3

Reply:
Thank you.